rtcpeerconnection.js

```diff
@@ -525,7 +538,9 @@ PeerConnection.prototype.handleOffer = function (offer, cb) {
             self._checkRemoteCandidate(line);
         }
     });
-    self.pc.setRemoteDescription(new RTCSessionDescription(offer),
+    self.pc.setRemoteDescription(
+        new RTCSessionDescription(offer)
+    ).then(
         function () {
             cb();
```

Comment:
i think we can just pass cb here

Reply:
I find this highly confusing:
```
.then(cb, cb)
```
it works because the success has arguments and the error uses the first one but...